### ISSUE TITLE
[Backend] Complete the Requirements Relating to the Guest and Admin

### ISSUE THREAD

**Issue by ahmetkudu** [bounswe/bounswe2023group6#550]
### Problem

Complete requirements below that have not been done yet:

 * 1.1.5.1.1 Guests shall be able to view the profiles of registered users. 
 * 1.1.4.1 Admins shall be able to see reports with the related forums. 
 * 1.1.4.2 Admins shall be able to see reports with the related LFGs. 
 * 1.1.4.3 Admins shall be able to see reports with the related game pages. 
 * 1.1.4.4 Admins shall be able to ban users. 
 * 1.1.4.6 Admins shall be able to apply any kind of necessary create/update/delete operation to the contents of the platform.

### Reviewers

Ömer Bahadıroğlu

### Deadline

19.12.2023 - 22.00

**Comment by ahmetkudu** [comment]
1.1.5.1.1 Guests shall be able to view the profiles of registered users. (completed in my previous issue [#559](https://github.com/bounswe/bounswe2023group6/issues/559) )

**Comment by ahmetkudu** [comment]
1.1.4.3 Admins shall be able to see reports with the related game pages. (completed)

Admin Controller:
![Screenshot 2023-12-19 at 00 18 59](https://github.com/bounswe/bounswe2023group6/assets/74921213/7edd572d-04a9-428a-a968-942790602d99)

Game Service:
![Screenshot 2023-12-19 at 00 19 51](https://github.com/bounswe/bounswe2023group6/assets/74921213/736a493d-469d-4dae-93cb-13a0c3e5f557)

**Comment by ahmetkudu** [comment]
1.1.4.2 Admins shall be able to see reports with the related LFGs. (not could not completed because LFGs endoints have not been ready yet.)

**Comment by ahmetkudu** [comment]
1.1.4.1 Admins shall be able to see reports with the related forums. (completed)

Admin Controller:
![Screenshot 2023-12-19 at 14 10 28](https://github.com/bounswe/bounswe2023group6/assets/74921213/5723f38c-e115-4769-93a0-7d53695c6ae8)

Post Service:
![Screenshot 2023-12-19 at 14 10 45](https://github.com/bounswe/bounswe2023group6/assets/74921213/e18c2577-1661-4d3f-9656-117931a5a636)

**Comment by ahmetkudu** [comment]
1.1.4.4 Admins shall be able to ban users. (completed)

Admin Controller:
<img width="1438" alt="Screenshot 2023-12-19 at 15 31 58" src="https://github.com/bounswe/bounswe2023group6/assets/74921213/d1c63a35-94ef-45cf-89c9-d12752c0388c">

User Service:
<img width="1435" alt="Screenshot 2023-12-19 at 15 34 19" src="https://github.com/bounswe/bounswe2023group6/assets/74921213/3eeb465a-6dc2-4c33-b939-692fb540957b">

Post Service:
<img width="1433" alt="Screenshot 2023-12-19 at 15 35 45" src="https://github.com/bounswe/bounswe2023group6/assets/74921213/decf80f2-7748-4fcd-b1fc-a0abc13f9444">

Game Service:
<img width="1427" alt="Screenshot 2023-12-19 at 15 36 27" src="https://github.com/bounswe/bounswe2023group6/assets/74921213/104aa2de-61a8-495e-be71-53c8c97fcc8e">

**Comment by ahmetkudu** [comment]
1.1.4.6 Admins shall be able to apply any kind of necessary create/update/delete operation to the contents of the platform.  (completed) 

We have decided that updating and creating the content is not appropriate, but deleting is enough. Also I added the operation to cancel reported game or post when the admin thinks there is no problem with them.  

Admin Controller:
![Screenshot 2023-12-19 at 19 28 56](https://github.com/bounswe/bounswe2023group6/assets/74921213/e26a9a39-54da-4ffe-9291-263b4338b134)

User Service:
![Screenshot 2023-12-19 at 19 44 36](https://github.com/bounswe/bounswe2023group6/assets/74921213/ae82a6a1-3b5d-44d4-8318-5bd8ef344666)
